### PR TITLE
MM-53729: Missing handlers

### PR DESCRIPTION
**Pull request by agarciamontoro**
#### Summary
This PR aims to fix the missing handlers ~`getSystemPing`~ and `getProfileImage`.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-53729

### CODE REVIEW

**Comment by agarciamontoro** [comment]
I've ran a quick test with this branch and the frequency of `getProfileImage` has increased from 0.29230 to 0.735. Definitely not the increase we need, but this handler is not even anymore in the tickets we have in the epic (the PR predates the new analysis), so I think we should merge, since it's better and that we are indeed stressing the 304 status code:
image
The average frequency broken down by status code is:
- 304: 0.429
- 200: 0.306
In any case, it's ready for review now :)

**Review by agnivade — APPROVED** [review]
One step closer to reality!

File: loadtest/user/user.go [hunk 1]
```diff
@@ -93,11 +93,9 @@ type User interface {
 
 	// SetProfileImage sets the profile image for the user.
 	SetProfileImage(data []byte) error
-	// GetProfileImage fetches the profile image for the user.
-	GetProfileImage() error
```
[the remark this answers is not in the document]
Comment (agarciamontoro) [reply loadtest/user/user.go:97]: Yup, it was the same as `GetProfileImageForUser` but only for the current user, and it was not used anywhere else.

**Comment by streamer45** [comment]
I don't see any mention to `getSystemPing`? Have we scratched that one?

**Comment by agarciamontoro** [comment]
Yes! Forgot to update the description. Copypasting the message from the channel:
> the way I implemented it, the frequency of `getSystemPing` will change depending on the number of users, but we want a steady rate independent of users (to simulate k8s hitting that endpoint). I'm leaning towards removing that change, which is not that relevant, and merge only the improvements to `getProfileImage`

**Review by streamer45 — APPROVED** [review]
Looks good!

File: loadtest/store/memstore/store.go [hunk 2]
```diff
@@ -962,27 +962,27 @@ func (s *MemStore) SetLicense(license map[string]string) error {
 
 // ProfileImage returns whether the profile image for the given user has been
 // stored.
```
[the remark this answers is not in the document]
Comment (agarciamontoro) [reply loadtest/store/memstore/store.go]: Ah, good catch, thanks!

**Pull request merged**